src/plugins/Workers/TransactionSyncStreamWorker/methods/syncUpToTheGapLimit.js

```diff
@@ -1,5 +1,19 @@
 const logger = require('../../../../logger');
 
+
+async function getBlockHeight(header) {
+  const prevHash = header.prevHash.reverse().toString('hex');
+
+  const prevBlock = await this.transport.getBlockByHash(prevHash);
+  try {
+    const prevBlockHeight = prevBlock.transactions[0].extraPayload.height;
+    return prevBlockHeight + 1;
+  } catch (e) {
+    const prevBlockHeight = prevBlock.transactions[1].extraPayload.height;
+    return prevBlockHeight + 1;
+  }
+}
+
```

Comment:
This is basically a copy of 
https://github.com/dashevo/wallet-lib/blob/7722d817373ae332d15641d0b3b73b9556e48a6c/src/transport/DAPIClientTransport/utils/getHeightFromMerkleBlockBuffer.js#L5-L18

would it be useful as a method on transport?